ofborg-eval-lib-tests failure looks unrelated

```
error: builder for '/nix/store/38k9b3azmz4s2k02301fbcbfaqhlykn5-aws-sdk-cpp-1.11.118.drv' failed with exit code 2;
       last 10 log lines:
       > [ 90%] Linking CXX executable aws-cpp-sdk-core-tests
       > 1.11.118
       > Segmentation fault
       > make[2]: *** [tests/aws-cpp-sdk-core-tests/CMakeFiles/aws-cpp-sdk-core-tests.dir/build.make:968: tests/aws-cpp-sdk-core-tests/aws-cpp-sdk-core-tests] Error 1
       > make[2]: *** Deleting file 'tests/aws-cpp-sdk-core-tests/aws-cpp-sdk-core-tests'
       > make[1]: *** [CMakeFiles/Makefile2:442: tests/aws-cpp-sdk-core-tests/CMakeFiles/aws-cpp-sdk-core-tests.dir/all] Error 2
       > make[1]: *** Waiting for unfinished jobs....
       > [ 91%] Linking CXX shared library libaws-cpp-sdk-s3.so
       > [ 91%] Built target aws-cpp-sdk-s3
       > make: *** [Makefile:136: all] Error 2
       For full logs, run 'nix log /nix/store/38k9b3azmz4s2k02301fbcbfaqhlykn5-aws-sdk-cpp-1.11.118.drv'.
       ```

--- a/pkgs/build-support/setup-hooks/separate-debug-info.sh
+++ b/pkgs/build-support/setup-hooks/separate-debug-info.sh
@@ -21,6 +21,9 @@ _separateDebugInfo() {
     while IFS= read -r -d $'\0' i; do
         if ! isELF "$i"; then continue; fi
 
+        [ -z "${READELF:-}" ] && echo "_separateDebugInfo: '\$READELF' variable is empty, skipping." 1>&2 && break
+        [ -z "${OBJCOPY:-}" ] && echo "_separateDebugInfo: '\$OBJCOPY' variable is empty, skipping." 1>&2 && break
+
         # Extract the Build ID. FIXME: there's probably a cleaner way.
         local id="$($READELF -n "$i" | sed 's/.*Build ID: \([0-9a-f]*\).*/\1/; t; d')"
         if [ "${#id}" != 40 ]; then

